code/modules/projectiles/ammunition/boxes.dm

```diff
@@ -388,41 +382,41 @@
 
 ///////// 7.62mm /////////
 
-/obj/item/ammo_magazine/c762
+/obj/item/ammo_magazine/m762
 	name = "magazine (7.62mm)"
-	icon_state = "c7.62"
+	icon_state = "m762-small"
 	mag_type = MAGAZINE
 	caliber = "a762"
 	matter = list(DEFAULT_WALL_MATERIAL = 2000)
 	ammo_type = /obj/item/ammo_casing/a762
 	max_ammo = 10
 	multiple_sprites = 1
 
-/obj/item/ammo_magazine/c762/ap
+/obj/item/ammo_magazine/m762/ap
 	name = "magazine (7.62mm armor-piercing)"
 	ammo_type = /obj/item/ammo_casing/a762/ap
 
-/obj/item/ammo_magazine/c762/empty
+/obj/item/ammo_magazine/m762/empty
 	initial_ammo = 0
 
-/obj/item/ammo_magazine/s762 // 's' for small!
+/obj/item/ammo_magazine/m762m // Intentionally not a subtype of m762 because it's supposed to be incompatible with the Z8 Bulldog rifle.

```

Comment:
This was originally done for balance reasons. I don't disagree with you but I didn't want players jamming 20 round mags into the bulldog.